### ISSUE TITLE
DAIR-V2X dataset download problem

### ISSUE THREAD

**Issue by Yolan1013** [ADLab-AutoDrive/BEVHeight#31]
Thank you for your excellent work!
But I met the problem when I try to download dataset form the official website. How can I request the data access? 
Any help would be appreciated.
![Screenshot from 2023-09-04 19-57-17](https://github.com/ADLab-AutoDrive/BEVHeight/assets/34823398/ccc9ae8f-b0d0-43c5-a7e2-5caaa90221b6)

**Comment by yanglei18** [comment]
DAIR-V2X-I is accessible only in China. By the way, multiple attempts are necessary.

**Comment by Yolan1013** [comment]
Thanks for your reply! I will try more times.

**Comment by hrysj2016** [comment]
I've tried many times, but I still haven't resolved it. Can you tell me how many times you tried until the website become avilable

**Comment by Yolan1013** [comment]
I haven't got the data link yet. I sent an email to dair@air.tsinghua.edu.cn (got on the website), but it's still the same.

> I've tried many times, but I still haven't resolved it. Can you tell me how many times you tried until the website become avilable

**Comment by hrysj2016** [comment]
> I haven't got the data link yet. I sent an email to [dair@air.tsinghua.edu.cn](mailto:dair@air.tsinghua.edu.cn) (got on the website), but it's still the same.
> 
> > I've tried many times, but I still haven't resolved it. Can you tell me how many times you tried until the website become avilable

Yes, I have the same issue. Besides posting here, I also sent a private message to the author on Zhihu, but have not received a reply. Once I get the download link, I will share it here.

**Comment by yanglei18** [comment]
It's OK Now!

**Comment by marmoh2002** [comment]
> It's OK Now!

have you been able to download it?

**Comment by DOUDOUZXL** [comment]
> > I haven't got the data link yet. I sent an email to [dair@air.tsinghua.edu.cn](mailto:dair@air.tsinghua.edu.cn) (got on the website), but it's still the same.
> > > I've tried many times, but I still haven't resolved it. Can you tell me how many times you tried until the website become avilable
> 
> Yes, I have the same issue. Besides posting here, I also sent a private message to the author on Zhihu, but have not received a reply. Once I get the download link, I will share it here.

Have you successfully obtained this data? If so, could you please share it?
